src/methods/scgpt_fine_tuned/config.vsh.yaml

```diff
@@ -0,0 +1,50 @@
+__merge__: ../../api/base_method.yaml
+
+name: scgpt_fine_tuned
+label: scGPT fine tuned
+summary: "Cell-type annotation by fine-tuning on a pre-trained scGPT model."
+description: |
+  scGPT is a foundation model for single-cell biology based on a generative pre-trained transformer and trained on a repository of over 33 million cells. Here, we fine-tune a pre-trained model on a reference dataset using the hyper-parameter recommenations for the cell-type task and then infer cell-types for the unlabelled cells in a query dataset.
+    
+references: 
+  doi: 10.1038/s41592-024-02201-0
+
+links: 
+  documentation: https://scgpt.readthedocs.io/en/latest/
+  repository: https://github.com/bowang-lab/scGPT
+
+info:
+  preferred_normalization: log_cp10k
```

Comment:
Should we give the normalised data or is it better to give counts and use scGPT's normalisation?

Reply:
Actually, I think the counts are used anyway

Reply:
Yeah, it uses counts.